queries.sql

```diff
@@ -15,3 +15,30 @@ SELECT * FROM animals WHERE neutered = 'true';
 SELECT * FROM animals WHERE animal_name != 'Gabumon';
 
 SELECT * FROM animals WHERE weight_per_kg >= '10.4' AND weight_per_kg <= '17.3';
+
+
+/* Transaction Update 1 */
+UPDATE animals SET species = 'Unspecified'
+ROLLBACK;
+
+
+/* Transaction Update 2 set species name to 'digimon' */
+UPDATE animals SET species = 'pokemon' WHERE species != 'digimon';

```

Comment:
Please UPDATE the animals table by setting the species column to pokemon for all animals that have species set to NULL.

Reply:
Thanks, changes implemented